<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Calibri&quot;;
background-color: rgb(141, 193, 231)

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="time_and_date_label">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>50</y>
      <width>150</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 153, 255);
border-radius:20px;
font-size:25px;
</string>
    </property>
    <property name="text">
     <string>1/1/2022
12:35</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="setting_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 153, 255);
border-radius:50px;
font-size:20px;
</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="top_widget">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="widget_section">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>1300</width>
      <height>610</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <action name="actiondfg">
   <property name="text">
    <string>dfg</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
